_posts/2021-05-22-fluxv2-openfaas.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "See how Flux's Git Ops ToolKit can simplify delivery"
```

Comment:
Should be:

`Flux's GitOps Toolkit` or perhaps "The Flux GitOps Toolkit" may read better.

Could you find a way to write the title as problem / benefit?

One of the points that @MrSimonEmms brought up was that v1 is now considered to be in maintenance mode, so it feels to me like have an opportunity to guide people to the new solution and help them see how to migrate.

"Learn how to migrate your OpenFaaS functions to The GitOps Toolkit"
"Upgrade your OpenFaaS deployments to Flux v2"